internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,69 @@
 package daysteps
 
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	slice := strings.Split(datastring, ",")
+	if len(slice) != 2 {
+		return fmt.Errorf("передано менее двух значений для метода Parse DaySteps")
+	}
+
+	steps, err := strconv.Atoi(slice[0])
+	if steps <= 0 {
+		return fmt.Errorf("значение шагов должно быть больше нуля")
+	}
+
+	if err != nil {
+		return fmt.Errorf("ошибка парсинга шагов")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
```suggestion
		return fmt.Errorf("conversion error: %w", err)
```